docs/developers/documentation/index.md

```diff
@@ -175,41 +202,47 @@ also be caught when previewing the built documentation. If your documentation
 change does not include code, you only need the napari docs dependencies
 installed.
 
-There are two ways you can preview the documentation website, by building
-locally or downloading the GitHub Actions built documentation when you submit
+There are two ways you can preview the documentation website: by building
+locally, or downloading the GitHub Actions built documentation when you submit
 your pull request.
 
-To build the documentation locally, run `make docs` from the root of
-the `napari` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
+To build the documentation locally, run `make docs` from the root of your local
+clone of the `napari/docs` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
 
 ```bash
 make docs
 ```
 
-The rendered HTML will be placed in `napari/docs/_build`. Find `index.html` in this folder and drag it
+```{note}
+The `make docs` command above assumes your have a local clone of the
+[`napari/napari`](https://github.com/napari/napari) repo at the same level as
+the `napari/docs` clone.
+```
+
+The rendered HTML will be placed in `docs/_build`. Find `index.html` in this folder and drag it
 into a browser to preview the website with your new document.
 
-Alternatively, When you submit your pull request, napari continuous integration
+Alternatively, when you submit your pull request, napari continuous integration
 includes a GitHub action that builds the documentation and saves the artifact
 for you to download. This is another way to check that your built documentation
 looks as you expect. To download the built documentation, go to your PR, scroll
 down to the continuous integration tests, then:
 
-1. click on 'details' next to 'Build Docs / Build & Upload Artifact (pull_request)'
+1. Click on **Details** next to `Build PR Docs / Build & Upload Artifact (pull_request)`:
```

Comment:
Outside scope of this PR, but with your docs build comment addition, I am wondering if this should be amended or removed.